assets/src/dashboard/app/views/editorSettings/googleAnalytics/index.js

```diff
@@ -204,12 +181,11 @@ function GoogleAnalyticsSettings({
 GoogleAnalyticsSettings.propTypes = {
   handleUpdate: PropTypes.func,
   googleAnalyticsId: PropTypes.string,
-  siteKitCapabilities: PropTypes.shape({
-    analyticsModuleActive: PropTypes.bool,
-    canActivatePlugins: PropTypes.bool,
-    canInstallPlugins: PropTypes.bool,
-    siteKitActive: PropTypes.bool,
-    siteKitInstalled: PropTypes.bool,
+  siteKitStatus: PropTypes.shape({
+    installed: PropTypes.bool,
+    active: PropTypes.bool,
+    analyticsActive: PropTypes.bool,
+    link: PropTypes.string,
   }),
```

Comment:
big fan of this update to the structure of these props.

Reply:
same, a lot better 👍